src/composables/useUtilities.ts

```diff
@@ -1,3 +1,5 @@
+/** @format */
+
 import { ref, Ref, computed, watchEffect } from 'vue'
```

Comment:
Sorry, my editor left that.

```suggestion
import { ref, Ref, computed, watchEffect } from 'vue'
```